samples/ChangefeedMigrationSample/Program.cs

```diff
@@ -0,0 +1,193 @@
+﻿namespace ChangefeedMigrationSample
+{
+    using DocumentDB.ChangeFeedProcessor;
+    using Microsoft.Azure.Documents;
+    using Microsoft.Azure.Documents.Client;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    // ------------------------------------------------------------------------------------------------
+    // This sample demonstrates using change processor library to read changes from source collection 
+    // to destination collection 
+    // ------------------------------------------------------------------------------------------------
+
+    public class DeviceReading
+    {
+        [JsonProperty("id")]
+        public string Id;
+
+        [JsonProperty("deviceId")]
+        public string DeviceId;
+
+        [JsonConverter(typeof(IsoDateTimeConverter))]
+        [JsonProperty("readingTime")]
+        public DateTime ReadingTime;
+
+        [JsonProperty("metricType")]
+        public string MetricType;
+
+        [JsonProperty("unit")]
+        public string Unit;
+
+        [JsonProperty("metricValue")]
+        public double MetricValue;
+    }
+
+    class Program
+    {
+        private const string EndPointUrl = "https://interntest.documents.azure.com:443/";
+        private const string PrimaryKey = "mXBmwssUDqDNL03M0qkmMBYizwpeIrLqCyFNUOGQsGCEeLRRkWJDleEORnVNzfQ13dkiIyxfhgVM4QAQLzQQzg==";
```

Comment:
It looks like you are trying to check in a real key here.  You need to be very careful with sample that you dont reveal information like this.  Be sure to rotate the keys now in the actual account.